client/src/components/DownloadButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { pdf } from '@react-pdf/renderer';
+import { MedInfoDocument } from './MedInfoDocument';
+import { saveAs } from 'file-saver';
+import { Button } from './ui/button';
+import { File } from 'lucide-react';
+import { Medication } from 'data';
+
+type Props = {
+  medications: Medication[];
+};
+
+export function DownloadButton({ medications }: Props) {
+  async function downloadPdf() {
+    const fileName = 'test.pdf';
```

Comment:
You should probably change this to a more realistic file name. `medicationList` or something